frontend/public/components/utils/resource-log.jsx

```diff
@@ -114,6 +144,28 @@ export class ResourceLog extends SafetyFirst {
     saveAs(blob, `${filename}.log`);
   }
 
+  // Toggle currently displayed log content to/from fullscreen
+  _toggleFullscreen () {
+    const logConsole = document.querySelector(".log-console");
+    if (!logConsole) {
+      return;
+    }
+
+    if (!isFullscreen()) {
+      this.requestFullscreen.call(logConsole);
+    } else {
+      if(document.exitFullscreen) {
+        document.exitFullscreen();
+      } else if(document.mozCancelFullScreen) {
+        document.mozCancelFullScreen();
+      } else if(document.webkitExitFullscreen) {
+        document.webkitExitFullscreen();
+      } else if (document.msExitFullscreen) {
+        document.msExitFullscreen();
```

Comment:
`document.msExitFullscreen();` may not be necessary since we don't officially support IE.

Reply:
Prob if we drop that it would be good to ensure it doesn't error/fall over.  No need to deliberately break. 😄